### PR TITLE
Updates to figures 1,3,4

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose

- Updates to figure 1, table 1 & 2 legends (add abbreviations)
- Update ordering / text for figure 3
- Update text for figure 4 to match new figures

### Issue

#203 
#237 

#### Which areas should receive a particularly close look?
double check updates

#### Is there anything that you want to discuss further?
no - will continue in another PR bc this has become long!

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

File: build/assets/custom-dictionary.txt [hunk 1]
```diff
@@ -60,9 +60,10 @@ Becher
 bedtools
 benchmarking
 Bendel
+Bevacizumab
 Bioconductor
 bioinformatics
-biorender
+BioRender
```
Comment (sjspielman) [inline build/assets/custom-dictionary.txt:66]:
```suggestion
Biorender
```

File: content/05.administrative.md [hunk 2]
```diff
@@ -19,4 +19,13 @@ CSG's spouse was an employee of Alex's Lemonade Stand Foundation, which was a sp
...
 ## Figure Titles and Legends
-**Figure 1. Overview of the OpenPBTA Project.**
\ No newline at end of file
+**Figure 1. Overview of the OpenPBTA project.**
+
+## Table Titles and Legends
+**Table 1. Molecular subtypes generated through the OpenPBTA project.**
+Listed are broad tumor histologies, molecular subtypes generated, and number of specimens subtyped within the OpenPBTA project.
+
+**Table 2. Patients with hypermutant tumors.**
```
Comment (sjspielman) [inline content/05.administrative.md:28]: Noting that I don't think this table has been created yet

**Review by sjspielman — APPROVED** [review]
Looks good! We may need to create Table 2 though, unless I am not seeing it somewhere?

Just one suggestion to fix the spelling error!

**Comment by jharenza** [comment]
> Looks good! We may need to create Table 2 though, unless I am not seeing it somewhere?

Not entirely created - you created a portion [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/b9a7a7f41695bf0bce9579a4a954cdeed4acefcc/analyses/mutational-signatures/08-explore_hypermutators.Rmd#L128-L148), but I realized there was a cell line instead of tumor in that. This is the same samples from [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/b9a7a7f41695bf0bce9579a4a954cdeed4acefcc/analyses/mutational-signatures/08-explore_hypermutators.Rmd#L168-L176), so we can definitely create in that notebook. The caveats are just renaming column names and that `Therapy post-biopsy` and the predispositions will have to be manually added and edited, respectively, because I had to look them up in another database and from path reports. I can create a ticket.


> Just one suggestion to fix the spelling error!

Thanks!

**Pull request merged**